boards/arm/frdm_k64f/Makefile.board

```diff
@@ -1,8 +1,5 @@
-FLASH_SCRIPT = openocd.sh
-DEBUG_SCRIPT = openocd.sh
-
-OPENOCD_LOAD_CMD = "flash write_image erase ${O}/${KERNEL_BIN_NAME} ${CONFIG_FLASH_BASE_ADDRESS}"
-OPENOCD_VERIFY_CMD = "verify_image ${O}/${KERNEL_BIN_NAME} ${CONFIG_FLASH_BASE_ADDRESS}"
+FLASH_SCRIPT = pyocd.sh
+DEBUG_SCRIPT = pyocd.sh
 
```

Comment:
I didn't change the default to pyocd earlier because I was hoping to do this for all the freedom boards at once to make them consistent. However, this depends on mbedmicro/pyOCD#302 which hasn't moved in a while. I'm ok to go ahead and change the default for this board.